v2/thirdpartyemailpassword/common-customizations/embed-sign-in-up-form.mdx

```diff
@@ -53,7 +53,218 @@ If you navigate to `/auth`, you should not see the widget anymore.
 
 ## Step 2: Render the component yourself
 
```

Comment:
Step 1 in this doc is only required for "Rendering the Auth Widget in a page" cause all other methods should also have the default auth route UI there. Usually during popup or during conditional rendering, users also have a full page auth UI. You should mention this explicitly in the popup and conditional rendering methods.

Reply:
Moved that inside the Case 1.